docs/guides/getting-started/opening-the-app.mdx

```diff
@@ -53,6 +53,17 @@ npm run cypress:open
 
 ...and Cypress will open right up for you.
 
+:::caution
+
+<strong>Best Practice</strong>
+
+Don't use `cypress` as the exact name of a script, especially if you use Yarn as package manager.
+Yarn will select the script in preference to the Cypress binary of the same name and
```

Comment:
I think removing this bit makes it more succinct and clear what the alert is all about. 

```suggestion
Yarn will select the script to the Cypress binary of the same name and
```

Reply:
@jaffrepaul 

Your suggestion changes the meaning, so I wouldn't want to agree to it.

What I'm trying to explain is that when there is a script and a binary with the same name `yarn run` (Yarn Classic and Modern) selects the script not the binary.

The Yarn Classic documentation doesn't explain this, however [Yarn Modern > `yarn run`](https://v3.yarnpkg.com/cli/run#details) says:

![image](https://github.com/cypress-io/cypress-documentation/assets/66998419/bc14a93c-ddb9-4153-ba42-6b58171c89e8)

If you have a better way of explaining this succinctly then let me know. It's always a challenge to write so that other people will understand, so I appreciate your taking a look at this!

Reply:
Another possibility would be to drop the detailed explanation and just say:

... otherwise Cypress CLI commands may not work as expected.

Reply:
Ah, I read this pretty quickly and while I understood the goal, I had them reversed. The more succinct version works but I think it's good to explain this in an issue with yarn and not cypress. 

This is even more verbose but we could try something like:

Do not use cypress as the exact name of a script, especially if you use Yarn as package manager. When running commands on the Cypress binary(e.g `yarn cypress verify`), Yarn will reference the script of the same name instead and Cypress CLI commands may not work as expected. Use instead a descriptive and non-ambiguous script name such as cypress:open or cypress:run.

Reply:
Thank you! I've used your text which explains the issue in an understandable way.